featurebyte/routes/feature_store/api.py

```diff
@@ -164,6 +177,9 @@ async def list_databases_in_feature_store(
         List databases
         """
         controller = request.state.app_container.feature_store_controller
+        feature_store = await FeatureStoreRouter.try_retrieve_feature_store(
+            controller, feature_store
+        )
         result: List[str] = await controller.list_databases(
```

Comment:
Curious what is the reason to retrieve the feature store?

Reply:
For older clients, the `database_details` field becomes empty when retrieving FeatureStoreModel from the service due to the change in `database_details` schema.  This will re-populate saved feature stores with this details on the service end to maintain backward compatibility. For unsaved feature stores this will not be a problem since the old schema provided by the client will still work.